### PR TITLE
Small Feature: debugbar link

### PR DESCRIPTION
**Pull request by enix-app**
I think this useful when we would test or check an URI by click the link without write or copy/paste to address bar. Only GET method.
Support custom placeholder.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Comment by MGatner** [comment]
This looks pretty cool but I’m not entirely sure from the video what is happening. It turns the route rows into usable and editable links? Please provide a description along with the code, if you would. 

Also I don’t think we have formatting correction for JavaScript but please make sure it adheres to the style guide: `if(input[n]` should have a space `if (input[n]`

**Comment by enix-app** [comment]
Preview GIF

Peek 2020-06-07 03-05

**Comment by lonnieezell** [comment]
It took me a couple of minutes of before I took to this one, but I do kindof dig this feature. @enix-app if you can fix the style tweaks that MGatner asked, we'll merge.

**Comment by enix-app** [comment]
I modified, tested on old browser IE11.

File: system/Debug/Toolbar/Views/toolbar.js [hunk 1]
```diff
@@ -600,5 +601,61 @@ var ciDebugBar = {
...
+				event.preventDefault()
+				var inputArray = [], t = 0;
+				var input = event.target.querySelectorAll('input[type=text]'); 
+				var tpl = event.target.getAttribute('data-debugbar-route-tpl');
+				
+				for (var n = 0; n < input.length; n++) {
+					if (input[n].value.length > 0) inputArray.push(input[n].value);
+				}
+
+				if(inputArray.length > 0) {
```
Comment (MGatner) [inline system/Debug/Toolbar/Views/toolbar.js:651]: `if (inputArray.length > 0) {`

File: system/Debug/Toolbar/Views/toolbar.js [hunk 2]
```diff
@@ -600,5 +601,61 @@ var ciDebugBar = {
...
+	},
+
+	routerLink: function() {
+		var row, _location;
+		var rowGet = document.querySelectorAll('#debug-bar td[data-debugbar-route="GET"]');
+		var patt = /\((?:[^)(]+|\((?:[^)(]+|\([^)(]*\))*\))*\)/;
+
+		for (var i = 0; i < rowGet.length; i++) {
+			row = rowGet[i];
+			if(!/\/\(.+?\)/.test(rowGet[i].innerText)) {
```
Comment (MGatner) [inline system/Debug/Toolbar/Views/toolbar.js:619]: `if (!/\/\(.+?\)/.test(rowGet[i].innerText)) {`

**Comment by enix-app** [comment]
OOT:

I can't login/register to forum website.

**Comment by MGatner** [comment]
Looks great, thank you!

**Comment by kenjis** [comment]
I'm not sure why, but this does not work now.
See https://github.com/codeigniter4/CodeIgniter4/issues/7324#issuecomment-1478830940

**Pull request merged**